src/ProjectService.Models.Dto/Models/ImageContent.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace LT.DigitalOffice.ProjectService.Models.Dto.Requests
```

Comment:
the same as 
namespace LT.DigitalOffice.UserService.Models.Dto.Requests.User
{
    public class AddImageRequest
    {
        public string Name { get; set; }
        public string Content { get; set; }
        public string Extension { get; set; }
    }
}